coffee-module/coffee-module-redis/src/main/java/hu/icellmobilsoft/coffee/module/redis/producer/JedisPoolProducer.java

```diff
@@ -66,7 +71,11 @@
     @RedisConnection(configKey = "")
     public JedisPool getJedisPool(InjectionPoint injectionPoint) {
         Optional<RedisConnection> annotation = AnnotationUtil.getAnnotation(injectionPoint, RedisConnection.class);
+        Optional<ConsumerPool> producerAnnotation = AnnotationUtil.getAnnotation(injectionPoint, ConsumerPool.class);
+        Optional<ProducerPool> consumerAnnotation = AnnotationUtil.getAnnotation(injectionPoint, ProducerPool.class);
         String configKey = annotation.map(RedisConnection::configKey).orElse(null);
+        configKey = producerAnnotation.isPresent() ? configKey + PRODUCER_POSTFIX : configKey;
+        configKey = consumerAnnotation.isPresent() ? configKey + CONSUMER_POSTFIX : configKey;
```

Comment:
Ha jól értem, akkor ez azt jeleti, hogy pl @ConsumerPool annotáció esetén az elvárt konfig ez lenne: `coffee.redis.{key}_consumer`, ha így van akkor erre kellene dokumentáció adoc-ba + az annotáció java doc-jába is jó lenne beleírni.

Reply:
Lehet egyszerűbb lenne új annotációk helyett a meglévő `@RedisConnection`-t bővíteni egy enummal pl poolType(?) (`DEFAULT,CONSUMER,PRODUCER`) és azt beépíteni a configba `coffee.redis.{configKey}.(default|consumer|producer).*`. Ha a stream-nel felmerült a külön pool, akkor valószínű felfog majd #137 ben is a pub/sub esetén, akkor ha az megint új annotáció, a végén el fogunk veszni a konfigolasban..

Reply:
kikerült